/**:
    ros__parameters:
        service_name: "/trigger_service"
        default_string: "No service available"
